dandiapi/api/views/zarr.py

```diff
@@ -176,15 +173,13 @@ def upload(self, request, zarr_id):
             if not self.request.user.has_perm('owner', zarr_archive.dandiset):
                 # The user does not have ownership permission
                 raise PermissionDenied()
-            logger.info(f'Beginning upload to zarr archive {zarr_archive.zarr_id}')
+            print(f'Beginning upload to zarr archive {zarr_archive.zarr_id}')
```

Comment:
What's the point of using `print` over `logger.info` here? As in, will this now only be visible locally?

Reply:
Oh, #1128 also changed those `print` statements to `logger.info`. That part shouldn't be reverted, I fixed it